pkg/logger/logger.go

```diff
@@ -13,10 +13,12 @@ func Get() *zap.Logger {
 	config.DisableStacktrace = true
 	config.InitialFields = map[string]any{"service": "sprayproxy"}
 	config.EncoderConfig.EncodeTime = utcRFC3339TimeEncoder
+	config.InitialFields = map[string]interface{}{"audit": "true"}
 	logger, err := config.Build()
 	if err != nil {
 		log.Fatalf("Failed to initialize zap logger: %v", err)
 	}
+	defer logger.Sync()
```

Comment:
That will sync the logger prematurely, as `Get()` is short living function. Instead we want to flush the logger on server exist and we are doing it here https://github.com/redhat-appstudio/sprayproxy/blob/main/pkg/server/server.go#L77 . So please remove that line.

Reply:
sure, that makes sense now.